introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello Sensei!
 
-Ruby
\ No newline at end of file
+Ruby
+
+* Git
+* Javascript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?